pkg/controller/imagebuild/component/builddispatcher.go

```diff
@@ -253,6 +256,13 @@ func (c *BuildDispatcherComponent) Reconcile(coreCtx *core.Context) (ctrl.Result
 	obj.Status.BuildTime = time.Since(start).Truncate(time.Millisecond).String()
 	buildSeg.End()
 
+	log.Info("Final image size: ", "imageSize", imageSize)
+	annotations := obj.GetAnnotations()
+	annotations["imagebuilder.dominodatalab.com/image-size"] = strconv.FormatInt(imageSize, 10)
+	obj.SetAnnotations(annotations)
```

Comment:
Trying to figure out why this annotation doesn't appear in https://github.com/dominodatalab/hephaestus/pull/131/files#diff-5c6176a5f8615022b3d6cd35ca34aaa3bc316e6c27af6c35037080ee2ff4142eR205

Reply:
annotations on the object in builddispatcher
```
{
    "imagebuilder.dominodatalab.com/build-id": "6597360e87e5c341bde190fd",
    "imagebuilder.dominodatalab.com/image-size": "3549772587",
    "imagebuilder.dominodatalab.com/last-image": "153827342122.dkr.ecr.us-west-2.amazonaws.com/richard31773/environment:65971a0e87e5c341bde190bb-9",
    "imagebuilder.dominodatalab.com/version": "v3"
}
```
in amqpmessenger
```
{
    "imagebuilder.dominodatalab.com/build-id": "6597360e87e5c341bde190fd",
    "imagebuilder.dominodatalab.com/last-image": "153827342122.dkr.ecr.us-west-2.amazonaws.com/richard31773/environment:65971a0e87e5c341bde190bb-9",
    "imagebuilder.dominodatalab.com/version": "v3"
}
```
both objects have the same uid and name
```
uid: "a3a70f97-7a90-46d8-b674-038bb17a4670",
name: "domino-build-65981a4dd9be8407b9c74e3b"
```

Reply:
1) Need to update structs
2) Need to run `make crds && make apply`